docs/_docs/features/debugger.md

```diff
@@ -6,7 +6,7 @@ permalink: /docs/features/debugger/
 ---
 
 One of the key features of Nuclide is its multiple-language debugging support. The Nuclide Debugger
-is provided within the familiar [Chrome DevTools](https://developer.chrome.com/devtools) interface.
+is provided with the familiar [Chrome DevTools](https://developer.chrome.com/devtools) interface.
```

Comment:
Ok - we can land this and think about whether we want to remove that sentence or not.